pkg/scheduler/api/job_info.go

```diff
@@ -707,6 +699,20 @@ func (ji *JobInfo) WaitingTaskNum() int32 {
 	return int32(len(ji.TaskStatusIndex[Pipelined]))
 }
 
+func (ji *JobInfo) PendingBestEffortTaskNum() int32 {
+	count := 0
+	for status, tasks := range ji.TaskStatusIndex {
+		if status == Pending {
+			for _, task := range tasks {
+				if task.InitResreq.IsEmpty() {
```

Comment:
Already has a `BestEffort` flag, don't need to calculate it any more.

Reply:
done